cmd/tendermint/commands/root.go

```diff
@@ -68,3 +69,10 @@ var RootCmd = &cobra.Command{
 		return nil
 	},
 }
+
+// deprecateSnakeCase is a util function for 0.34.1. Should be removed in 0.35
+func deprecateSnakeCase(cmd *cobra.Command, args []string) {
+	if strings.Contains(cmd.CalledAs(), "_") {
+		fmt.Println("Deprecated: snake_case commands will be replaced by hyphen-case commands in the next major release")
+	}
+}
```

Comment:
As this is going into master i.e. is therefore slated for 0.35, should I remove this now or let it stay for one more release cycle (i.e. remove this and all the snake_case aliases in 1.0)

Reply:
Since the work is already done, and it seems like the polite thing to do, I'd personally keep it around for 0.35. Don't think it's a big deal though.